packages/core/src/components/badge/badge.tsx

```diff
@@ -8,13 +8,14 @@ import clsx from 'clsx';
 })
 export class Badge {
   @Prop()
-  status?: 'primary' | 'success' | 'info' | 'warning' | 'danger';
+  status?: 'primary' | 'success' | 'info' | 'warning' | 'danger' | 'outline';
 
   private static statusMap = new Map<string, string>([
     ['success', 'Gelukt'],
     ['info', 'Opmerking'],
     ['warning', 'Waarschuwing'],
-    ['danger', 'Fout']
+    ['danger', 'Fout'],
+    ['outline', 'Rand']
```

Comment:
Zie opmerking https://github.com/dso-toolkit/dso-toolkit/pull/1567/files#r879584820
Verder, viel ons vanmiddag natuurlijk op dat 'primary' een lege sr-only `:` tekent, dat willen we natuurlijk ook niet.